### PR TITLE
remove ssh public key

### PR DESCRIPTION
**Pull request by Kuldip-Nanda**

### CODE REVIEW

**Review by agrare — APPROVED** [review]
:+1: 
@Kuldip-Nanda if this was your ssh key recommend deleting it and creating a new one

File: lib/tasks_private/terraform/id_rsa.pub [hunk 1]
```diff
@@ -1 +0,0 @@
-ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDMhEWdC/W2ceN4s2zrKMRLZYTnB1HfYlqSFrhc/p5D9HarpQgH4XVL9hWD8xEO5Izysuu3q8FbO322snzbSDKE3GAD8ek3bFZ1fLAz4WVBw+f5PKeK8HzwONS02oaZ9ZVG2kguaLjYcrWg6EbVW5myJdIhj8ugMZKRCrNs0DsEn1fjcH+AncLJRxnri6bb14+cmoXARQqtfOYl6opDMK8oHCxTPvvXa9YJykAKAbl0fMh7QIBZ8dOtP81/G0m15nqUX1nEohge2oX76u7AdZwgfXVmaaSo2L9vAnUcZCqBBsuHkjCvczQtMlO3uO+gYYC/wa2zzm94LdEcQ3O47qDzVfywaLckV61XRwkEq0BRqZT0nlwYvlW+UnG4wtwYiPVM7DiawQkEcz7Y1MMLGECPodraQq0CWWhyVZZ9oQH0f6BkHCEm9fwgicH42on8bmZHzHzyLSlw5JjI2tKDsqfKh9iTYnLINPG+KWrM6/02vqobDcdmpBT7G1NiiQVMO4k= root@localhost.localdomain
```
Comment (jaywcarman) [inline lib/tasks_private/terraform/id_rsa.pub:1]: @Kuldip-Nanda is this consumed anywhere? As far as I can tell it isn't.

**Comment by jaywcarman** [comment]
> +1 @Kuldip-Nanda if this was your ssh key recommend deleting it and creating a new one

It's just a public key, which doesn't need to be kept safe. For example, GitHub exposes all of the public keys you add to your account (https://github.com/jaywcarman.keys)

File: lib/tasks_private/terraform/id_rsa.pub (hunk 1 above)
Comment (Kuldip-Nanda) [reply lib/tasks_private/terraform/id_rsa.pub:1]: Nope, it is not consumed anywhere. A redundant file.

Comment (jaywcarman) [reply lib/tasks_private/terraform/id_rsa.pub:1]: Cool, was just going to suggest adding it to a .gitignore if it were used somewhere. LGTM

**Review by jaywcarman — APPROVED** [review]

File: lib/tasks_private/terraform/id_rsa.pub (hunk 1 above)
Comment (Fryguy) [reply lib/tasks_private/terraform/id_rsa.pub:1]: :+1: for .gitignore if it is required for this file to exist.

**Pull request merged**